modules/sample-ztp-custom-resources.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="sample-ztp-custom-resources_{context}"]
+= Sample ZTP custom resources
+
+Optional: You can use Zero touch provisioning (ZTP) custom resource (CR) objects to install {product-title} with the Agent-based Installer.
+
+You can customize the following ZTP custom resources to specify more details about your {product-title} cluster. The following sample ZTP custom resources are for a single-node cluster.
+
+**AgentClusterInstall.yaml**
+
+[source,yaml]
+----
+  apiVersion: extensions.hive.openshift.io/v1beta1
+  kind: AgentClusterInstall
+  metadata:
+    name: test-agent-cluster-install
+    namespace: cluster0
+  spec:
+    clusterDeploymentRef:
+      name: ostest
+    imageSetRef:
+      name: openshift-4.11
```

Comment:
We should use a docs variable here that points to the current release (4.12 now )